### DISCUSSION TITLE
Calssification

### DISCUSSION THREAD

**Discussion by kodohoi** [TorchIO-project/torchio#1078] [Q&A] [answered]
I want to do a classification task. However, because the shape of my data is 3 dimensions and they are large size, I cannot do mini-batch learning. 
Is it possible to do a classification task with TorchIO?
I'm considering using a patch-based pipeline in training to extract important features with WeightedSampler and evaluating test images which is full-size (not patched) with batch_size =1.
Is this okay?

**Comment by fepegar** [comment]
Hi, @kodohoi.

> Is it possible to do a classification task with TorchIO?

Yes. TorchIO is independent of the task.

> Is this okay?

Can you please elaborate? What do you need help with?

**Reply by kodohoi** [reply]
Thank you for your reply.

Because most medical images are large, patch-piped learning is implemented like Torchio. However, how should I interrupt the result of each patch for the classification task?
For example, in the case of binary classification like pneumonia or non-pneumonia, the trained model would output the probability of each patch. I want to know how to combine the result. Although all idea that I come up with is Multiple Instance Learning, is there any method to make the most of Torchio?

**Reply by fepegar** [reply, marked as answer]
MIL sounds good to me. But this is a machine learning question and out of the scope of TorchIO, so I won't be able to provide strong advice.
